lib/travis/build/script/julia.rb

```diff
@@ -51,7 +51,7 @@ def setup
             else
               sh.failure "Operating system not supported: #{config[:os]}"
             end
-            sh.cmd 'export PATH="${PATH}:${HOME}/julia/bin"'
+            sh.cmd 'export PATH="${PATH}:${TRAVIS_HOME}/julia/bin"'
```

Comment:
It doesn't look like `TRAVIS_HOME` is [an official environment variable](https://docs.travis-ci.com/user/environment-variables/) and it doesn't seem to be defined when I run the container locally.